### DISCUSSION TITLE
Mounting volumes

### DISCUSSION THREAD

**Discussion by Manu224781** [flatcar/Flatcar#1503] [Q&A]
Hey everybody!

I am currently working on a Flatcar deployment on Proxmox and VMware ESXi. So far, so good. I can install the VM automatically, and on boot, it's running an ignition for setup.

Now, I am wondering what the best practice is for the volumes. Do you mount an NFS share for them? I will be running PostgreSQL within Flatcar and have read that NFS shares are not the best option. Or do you simply mount a second HDD?

I'm asking because if anything goes wrong and I have to reinstall the VM, it's probably not ideal to have all the volumes on the only partition that gets deleted during reinstallation.

I am thankful for any tips.

**Comment by till** [comment]
Proxmox has Ceph to provision volumes but that's outside of flatcar.

It's also a lot of YMMV since the performance heavily depends on your hardware (disks, controllers, network, switches, etc.) and how Ceph is configured.

Either way - NFS or blockstorage: you need to supply the correct ignition to get them mounted.
